File: 백준/Bronze/1924. 2007년/2007년.py
result = ['MON', 'TUE', 'WED', 'THU', 'FRI', 'SAT', 'SUN']
day_per_month = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31]

x, y = map(int, input().split())
total_day = y

if x > 1:
    total_day += sum(day_per_month[:x - 1])

print(result[(total_day % 7) - 1])

File: 백준/Bronze/1924. 2007년/2007년.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int x = sc.nextInt();
        int y = sc.nextInt();
        String[] day = {"SUN", "MON", "TUE", "WED", "THU", "FRI", "SAT"};
        int[] month = {0, 31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 31};
        
        int days = y;
        for (int i = 1; i < x; i++) days += month[i];
        
        System.out.println(day[days % 7]);
    }
}